### PR TITLE
feat: Cleanup actor-builds, actor-runs

### PR DESCRIPTION
**Pull request by HonzaTuron**

### CODE REVIEW

File: openapi/components/schemas/actor-runs/Run.yaml [hunk 1]
```diff
@@ -49,15 +49,15 @@ properties:
     example: false
   meta:
     allOf:
-      - $ref: ../meta.yaml
+      - $ref: ../common/BuildsMeta.yaml
       - {}
   stats:
     allOf:
```
Comment (tobice) [inline openapi/components/schemas/actor-runs/Run.yaml:55]: (nit) `allOf` can be dropped

File: openapi/components/schemas/actor-runs/RunMeta.yaml
Comment (tobice) [inline openapi/components/schemas/actor-runs/RunMeta.yaml:1]: Can we merge `RunMeta` and `BuildsMeta`?

If not, why is `BuildsMeta` in `common`?

File: openapi/components/schemas/actor-runs/Run.yaml [hunk 2]
```diff
@@ -49,15 +49,15 @@ properties:
     example: false
   meta:
     allOf:
-      - $ref: ../meta.yaml
+      - $ref: ../common/BuildsMeta.yaml
```
Comment (tobice) [inline openapi/components/schemas/actor-runs/Run.yaml:52]: Why `BuildsMeta` when this is a run?

**Review by tobice — CHANGES_REQUESTED** [review]

File: openapi/components/schemas/actor-runs/Run.yaml (hunk 2 above)
Comment (HonzaTuron) [reply openapi/components/schemas/actor-runs/Run.yaml:52]: It was defined like this but it was probably bug. Fixed :)

File: openapi/components/schemas/actor-runs/RunMeta.yaml
Comment (HonzaTuron) [reply openapi/components/schemas/actor-runs/RunMeta.yaml:1]: Not possible to merge, moved to actor-builds :)

**Review by tobice — APPROVED** [review]

**Pull request merged**